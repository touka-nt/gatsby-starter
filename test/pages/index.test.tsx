import * as React  from 'react'
import { shallow } from 'enzyme'
import  { shallowToJson } from 'enzyme-to-json'
import IndexPage from 'src/pages'

describe('Sample', () => {
    test('say hello', () => {
        const wrapper = shallow(<IndexPage />)
        expect(shallowToJson(wrapper)).toMatchSnapshot()
    })
})