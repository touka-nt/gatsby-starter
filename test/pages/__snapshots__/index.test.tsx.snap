// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample say hello 1`] = `
<div>
  Hello, world.
</div>
`;
